schemas/common/telemetry/logger/v1.schema.json

```
@@ -0,0 +1,36 @@
+{
+  "$id": "https://mapcolonies.com/common/telemetry/logger/v1",
+  "title": "commonTelemetryLoggerV1",
+  "description": "Telemetry logger configuration",
+  "type": "object",
+  "default": {},
+  "required": ["level", "prettyPrint"],
+  "properties": {
+    "level": {
+      "$ref": "#/definitions/level"
+    },
+    "prettyPrint": {
+      "$ref": "#/definitions/prettyPrint"
+    }
+  },
+  "examples": [
+    {
+      "level": "info",
+      "prettyPrint": false
+    }
+  ],
+  "definitions": {
+    "level": {
+      "default": "info",
+      "description": "The log level",
+      "x-env-value": "LOG_LEVEL",
+      "enum": ["trace", "debug", "info", "warn", "error", "fatal"]
+    },
+    "prettyPrint": {
+      "default": false,
+      "description": "Whether to pretty print logs",
+      "type": "boolean",
+      "x-env-value": "LOG_PRETTY_PRINT"
```

```suggestion
      "x-env-value": "LOG_PRETTY_PRINT_ENABLED"
```
so we dont break API

---

done